internal/distro/fedora/distro.go

```diff
@@ -43,6 +43,9 @@ const (
 
 	//Kernel options for ami, qcow2, openstack, vhd and vmdk types
 	defaultKernelOptions = "ro no_timer_check console=ttyS0,115200n8 biosdevname=0 net.ifnames=0"
+
+	// Directory for saving openscap results
+	oscapDataDir = "/var/log/osbuild/openscap_data"
```

Comment:
Are we sure we want to save this to the image?

Reply:
Yeah sorry, didn't get the chance to push the changes earlier.

Reply:
Ah, sorry, misread the timestamps.